addons/web/static/src/webclient/actions/action_service.js

```diff
@@ -64,9 +64,15 @@ const viewRegistry = registry.category("views");
 /**
  * @typedef {Object} ActionOptions
  * @property {Context} [additionalContext]
+ * This parameter will be added to the action context, and remain consistent between loads.
```

Comment:
I don't know that I should actually add comments here, but I spent a truly embarassing amount of time trying to wrestle with the way the doAction method even functions.
I feel like having accessible documentation on this would help trendemously with understanding how to do complex stuff with it (or, well, any documentation at all).

Reply:
Also, while we're here: if I wanted to add precisions to the Owl doc, now would I go about doing that?